build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+set +o pipefail
+
+driver_images_tag_from_branch=develop
+DOCKER_HUB_USERNAME=$csiblock_dockerhub_username
+DOCKER_HUB_PASSWORD=$csiblock_dockerhub_password
```

Comment:
I think "docker registry" is better than "docker hub", not only because docker hub is usually one word, but because we might change registries in the future

so please rename wherever applicable

Reply:
done

Reply:
I still see ["docker hub"](https://github.com/IBM/ibm-block-csi-operator/blob/773eada2d3347f333b33d682a9af30a518985003/build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh#L5)

Reply:
this is an environment variable that [docker-hub command](https://libraries.io/pypi/docker-hub) requires

Reply:
why should we prefer a dockerhub-specific command over a generic docker command (e.g. [docker manifest inspect](https://stackoverflow.com/a/52077346))?

Reply:
IMO it's inefficient to "download" all tags and iterate over them anyway

Reply:
> why should we prefer a dockerhub-specific command over a generic docker command (e.g. [docker manifest inspect](https://stackoverflow.com/a/52077346))?
>
in `docker manifest inspect` you need a specific tag, no? I am trying to search for the right tag

Reply:
> IMO it's inefficient to "download" all tags and iterate over them anyway

>
I am not iterating over  all the tags, I extract them and with grep I filter them, so it iterates only on a few tags

Reply:
grep also "iterates", I meant that it's not efficient bandwidth-wise and CPU-wise

Reply:
> in `docker manifest inspect` you need a specific tag, no? I am trying to search for the right tag

I thought you know which tag you're looking for

Reply:
done

Reply:
done